### PR TITLE
Polished Magic.Link authentication docs

### PR DESCRIPTION
**Pull request by LBrian**
From my actual experience of setting up `magic.link` authentication in my RR projects, I believe with these pieces of information there in place at the beginning, it could save a lot of my time googling and jumping between `magic.link` and RR docs.

### CODE REVIEW

File: docs/authentication.md [hunk 1]
```diff
@@ -390,8 +389,8 @@ To get your application keys, go to [dashboard.magic.link](https://dashboard.mag
 > **Including Environment Variables in Serverless Deployment:** in addition to adding the following env vars to your deployment hosting provider, you _must_ take an additional step to include them in your deployment build process. Using the names exactly as given below, follow the instructions in [this document](https://redwoodjs.com/docs/environment-variables) to "Whitelist them in your `redwood.toml`".
 
 ```js
-// web/src/App.js
-import { AuthProvider } from '@redwoodjs/auth'
+// web/src/App.tsx
```
Comment (thedavidprice) [inline docs/authentication.md:392]:
```suggestion
// web/src/App.js|tsx
```

Can be either depending on a project's language target.

File: docs/authentication.md [hunk 2]
```diff
@@ -416,6 +415,23 @@ const App = () => (
 export default App
 ```
 
+```js
+// web/src/Routes.tsx
```
Comment (thedavidprice) [inline docs/authentication.md:419]:
```suggestion
// web/src/Routes.js|tsx
```

File: docs/authentication.md [hunk 3]
```diff
@@ -889,7 +905,30 @@ The Redwood API does not include the functionality to decode Magic.link authenti
 
 +++ View Magic.link Options
 
-Magic.link recommends using the issuer as the userID.
+##### Installation
+
+You must manually install the **Magic Admin SDK** first
```
Comment (thedavidprice) [inline docs/authentication.md:910]:
```suggestion
First, you must manually install the **Magic Admin SDK** in your project's `api/package.json`.
```

File: docs/authentication.md [hunk 4]
```diff
@@ -889,7 +905,30 @@ The Redwood API does not include the functionality to decode Magic.link authenti
...
+
+```terminal
+yarn workspace api add @magic-sdk/admin
+```
+
+##### Setup
+
+To get your application running _without setting up_ `Prisma`, get your `SECRET KEY` from [dashboard.magic.link](https://dashboard.magic.link/) then add them to your `.env`.
```
Comment (thedavidprice) [inline docs/authentication.md:918]:
```suggestion
To get your application running _without setting up_ `Prisma`, get your `SECRET KEY` from [dashboard.magic.link](https://dashboard.magic.link/). Then add `MAGICLINK_SECRET` to your `.env`.
```

File: docs/authentication.md [hunk 5]
```diff
@@ -889,7 +905,30 @@ The Redwood API does not include the functionality to decode Magic.link authenti
...
+
+```js
+// redwood/api/src/lib/auth.ts
```
Comment (thedavidprice) [inline docs/authentication.md:921]:
```suggestion
// redwood/api/src/lib/auth.js|ts
```

**Comment by thedavidprice** [comment]
@LBrian ha! you were right there with me on those suggestions! Let's get this in.

Thanks 🚀

**Comment by LBrian** [comment]
> @LBrian ha! you were right there with me on those suggestions! Let's get this in.
> 
> Thanks 🚀

Responsive is frontend dev's nature!! 😄

**Pull request merged**